### DISCUSSION TITLE
MacOS Monterey VirtualBox, Intel mac not working

### DISCUSSION THREAD

**Discussion by dstorozhuk** [docksal/docksal#1587] [General]
Looks like the virtual box is not supported by apple on MacOS Monterey even on Intel Mac? 

Related issues: 
https://forums.virtualbox.org/viewtopic.php?f=39&t=103438&sid=f87499d4ed999841ba5c606f202fb68b
https://forums.virtualbox.org/viewtopic.php?f=39&t=104265&sid=f87499d4ed999841ba5c606f202fb68b
https://forums.virtualbox.org/viewtopic.php?f=39&t=104272&sid=f87499d4ed999841ba5c606f202fb68b

**Comment by lmakarov** [comment]
There've always been issues with new macOS releases messing up development tools and workflows.

In this particular case, it's a matter of VirtualBox to release a version compatible with MacOS Monterey:

> Solution to this one, essentially, is that Monterey is not a supported host OS yet, and the devs are working on it. Monterey just went final release a few days ago, meaning Apple should theoretically have stopped twiddling with it. Now the devs can lock in on what they need to do to support Monterey. Same thing happened with Catalina and Big Sur.

https://forums.virtualbox.org/viewtopic.php?f=39&t=104272#p508362

**Comment by lmakarov** [comment]
This was addressed in https://github.com/docksal/docksal/pull/1622
